.github/workflows/release-create.yml

```diff
@@ -0,0 +1,147 @@
+# Copyright (C) 2020 Dremio
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+# Projectnessie GitHub Release workflow
+
+# Manually triggered workflow, takes the "release-version" and "next-version" arguments.
+
+# This workflow creates the git commits + git tag for a Nessie release.
+# It requires a fully successful CI status of the commit going to be released, i.e. we rely on
+# the "Main CI" workflow here.
+
+# When this workflow pushes the release tag (e.g. `nessie-0.5.1`), the `release-publish.yml`
+# workflow publishes the release artifacts
+
+# Projectnessie really prefers a linear git history - aka no merges. PRs must not be merged
+# while the release workflow runs. In case the git history would not be linear, this workflow will
+# fail without having uploaded/pushed any release artifacts.
+
+# No secrets (except the last step uses the GITHUB_TOKEN to push release-commits+tag to the branch)
+# are needed for this workflow.
+
+name: Create Release
+
+on:
```

Comment:
was there something here about it only being triggered by a maintainer?

Reply:
Yea - I've removed the `if actor in [...]`, because IIRC only project/repo-maintainers can manually start workflows. But even if someone would be able to start the workflow, the worst thing that could happen is that we can two commits + 1 tag. The actual deployment action (`release-publish.yml`) is protected via the GitHub repository, which would require a "review" (aka some human manually approving the workflow run in the `release` GitHub environment).